### DISCUSSION TITLE
关于开启单人游戏或加入服务器时游戏崩溃的提问

### DISCUSSION THREAD

**Discussion by umarurize** [Meloong-Git/PCL#1821] [提问] [answered]
如题

近期新组了电脑，可以正常进入游戏主界面。但是在进入单人世界或者加入服务器时就会闪退。
加入的mod在以前的电脑上可以正常游玩
错误报告见附件

求大佬帮忙！

[Uploading 错误报告-2023-5-11_19.07.24.zip…]()

**Comment by iPodtouch6557** [comment]
没传上来，重传一遍qwq
~还有新组一个电脑是啥意思，你自己组装了一个？~

**Reply by umarurize** [reply]
是的 是自己组装的itx小主机
[错误报告-2023-5-11_19.51.49.zip](https://github.com/Hex-Dragon/PCL2/files/11451821/-2023-5-11_19.51.49.zip)

**Reply by iPodtouch6557** [reply]
一般自组机会出现更多的问题……

**Comment by WForst-Breeze** [comment]


**Reply by umarurize** [reply]
[错误报告-2023-5-11_22.14.53.zip](https://github.com/Hex-Dragon/PCL2/files/11453121/-2023-5-11_22.14.53.zip)
这是添加后的错误报告

**Comment by wdnmd23333fdhsuidus** [comment]
来个crash-report（

**Comment by Pigeon0v0** [comment]
1. 检查你的网络连接
2. 尝试重新安装整合包和 Mod 文件
3. 尝试更换 [这里的 Java ](https://azul.com/downloads)

**Reply by Pigeon0v0** [reply]
> Java 链接应该是 https://azul.com/downloads 吧……
> 
> 麻烦改一下.jpg

这不是对的吗.jpg

**Comment by umarurize** [comment, marked as answer]
事隔多月，再回首当初的问题。我认为是使用AMD的问题，服内的玩家使用AMD也会出现玄学问题。自从换了N卡，这些问题就不再出现了。

当初的应急解决办法是使用 hmcl 的软渲染器启动，这样就用不到A卡，直接用cpu渲染，但是性能很差就是了。

玄学，应该是某些渲染类优化mod不兼容AMD。

**Reply by zkitefly** [reply]
可能是amd驱动问题，今年的驱动有点问题

如果可以；你可以安装这个amd驱动看看

https://www.amd.com/en/support/kb/release-notes/rn-rad-win-22-5-1 
![](https://img.kookapp.cn/assets/2023-06/VoQjjMQlTg0x905x.png)

**Reply by umarurize** [reply]
已经换上N卡啦 awa 不过感觉mc真的是A卡黑户，各种玄学问题
